modyn/protos/selector.proto

```diff
@@ -74,6 +76,14 @@ message GetNumberOfPartitionsRequest {
 
 message NumberOfPartitionsResponse { int32 num_partitions = 1; }
 
+message GetAvailableLabelsRequest {
```

Comment:
Could you please add an integration test for this :)?

Reply:
Added